mappings/c051258fa18739d7a4de37b2c3d232bd1bfb0d9c102bdafcf29ee202426c7565205472656573.json

```diff
@@ -52,11 +52,11 @@
         ]
     },
     "description": {
-        "sequenceNumber": 1,
-        "value": "Raising the bar on Cardano with some top shelf weed.",
+        "sequenceNumber": 2,
+        "value": "Mountains are just stoned trees.",
         "signatures": [
             {
-                "signature": "0ef3c1bbf5d86925c9473846343e84396117b6c4a38d13a7e72992a607cc2f171646a735f92e930e4e3656fc880678c84384085ac591031d4954e8cb2789ab05",
+                "signature": "34c3b46010748e836a319222589ea77cad75c795f576b57bab90a27a3ecfafa323d8774aa4931d9d4fe93d79172eb3caceab4a744c96081f5dbd9287c9dc8c0b",
```

Comment:
Hi @mrsqurriel, 

the validator is failing on the description signature. Can you please resign the file and submit it again? 

Thanks!